docs/content/config.md

```diff
@@ -24,14 +21,47 @@ workspaces are created by calling the appropriate function on the IWorkspaceCont
 context.WorkspaceContainer.CreateWorkspace("workspace");
 context.WorkspaceContainer.CreateWorkspace("another");
 context.WorkspaceContainer.CreateWorkspaces("more", "than", "one");
+```
+ 
+
+##  Layout Engines
+
+LayoutEngines define the way windows get arranged on each workspace. There are a number of different ways to layout windows and you can swap between them dynamically. By default workspacer loads 2 engines to switch between, the FullLayoutEngine and the TallLayoutEngine, howeveryou can change the set of default layout engines via:
+
+```csharp
+context.DefaultLayouts = () => new ILayoutEngine[] { new FullLayoutEngine() };
+```
+
+ You can even manually specify the LayoutEngine(s) on a per-workspace basis, by passing the different LayoutEngines to the `CreateWorkspace` (or equivalent) function
 
-# you can even define custom layouts for a specific workspace
+```csharp
 context.WorkspaceContainer.CreateWorkspace("layouts!", new FullLayoutEngine(), new TallLayoutEngine());
 ```
 
-## how do I make workspacer ignore some windows? <br/> how do I make some windows always go to a specific workspace?
 
-workspacer will ignore certain system windows, such as Task Manager and workspacer itself, and will by route workspaces to the focused workspace when they are opened. as with most everything else, this is completely customizable via IWindowRouter.
+### Existing LayoutEngines
+* [FullLayoutEnine](https://github.com/workspacer/workspacer/blob/master/src/workspacer.Shared/Layout/PaneLayoutEngine.cs)
+    * Maximizes the current focused window and hides all others
+* [TallLayoutEngine](https://github.com/workspacer/workspacer/blob/master/src/workspacer.Shared/Layout/PaneLayoutEngine.cs)
+    * Splits the screen in two horizontal zones, 1 primary and one secondary
+    * Windows get created in the secondary zone by default
+    * The number of windows in the primary zone can be dynamically adjusted
+ * [PaneLayoutEngines](https://github.com/workspacer/workspacer/blob/master/src/workspacer.Shared/Layout/PaneLayoutEngine.cs)
+    * Orders windows in either rows or columns depending on the setting
```

Comment:
I think it might be a bit clearer if `VertLayoutEngine` and `HorzLayoutEngine` are explicitly named - especially `PaneLayoutEngine` is an abstract class.

Reply:
good point, have addressed this in the new PR